ui/job-view/pushes/PushHeader.jsx

```diff
@@ -130,15 +130,20 @@ class PushHeader extends React.PureComponent {
   pinAllShownJobs() {
     const {
       selectedJob, setSelectedJob, pinJobs, expandAllPushGroups, getAllShownJobs,
+      notify,
     } = this.props;
     const shownJobs = getAllShownJobs(this.props.pushId);
```

Comment:
`this.props.pushId` is the only prop that isn't included in the destructuring above. Should it be? 

I guess we could either do so for all props, or else only ones that are used more than once? (But either way, it would be good to have consistency)

Reply:
Oh, heh.  Just a flub.  You're totally right.  Fixed...

Yeah, I feel like if I need more than one item from ``props`` or ``state``, then I destructure.  If it's just one, sometimes I just use dot notation it makes for less lines.